docker-compose.malware-scanner.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+
+  clamav:
+    image: clamav/clamav:stable_base
+    container_name: clamav
+    restart: on-failure
+    logging: *default-logging
+    volumes:
+      - ./docker/data/clamav/clamav/defs:/var/lib/clamav
+      - ./docker/clamav/clamd.conf:/etc/clamav/clamd.conf:ro
+    expose:
+      - 3310 # NEVER expose this outside of the local network!
+    deploy:
+      resources:
+        limits:
+          cpus: '0.50'
+    networks:
+      shared:
+        ipv4_address: 10.10.10.100
+
+  malware-scanner:
+    build:
+      context: ./docker/malware-scanner
+      dockerfile: Dockerfile
+      args:
+        branch: ivo/impl
+    container_name: malware-scanner
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
```

Comment:
what's the reason behind having both `env_file` and `environment` 
is that for being able to default certain values?

Reply:
From my perspective - yes, that's why I put those there. I am not sure whether we need to specifically spell out every single env var we're using. I see Karol did that for accounts and set this:
```
      - SKYNET_DB_HOST=${SKYNET_DB_HOST}
      - SKYNET_DB_PORT=${SKYNET_DB_PORT}
      - SKYNET_DB_USER=${SKYNET_DB_USER}
      - SKYNET_DB_PASS=${SKYNET_DB_PASS}
      - COOKIE_DOMAIN=${COOKIE_DOMAIN}
      - COOKIE_HASH_KEY=${COOKIE_HASH_KEY}
      - COOKIE_ENC_KEY=${COOKIE_ENC_KEY}
      - STRIPE_API_KEY=${STRIPE_API_KEY}
      - STRIPE_WEBHOOK_SECRET=${STRIPE_WEBHOOK_SECRET}
      - SKYNET_ACCOUNTS_LOG_LEVEL=${SKYNET_ACCOUNTS_LOG_LEVEL}
      - KRATOS_ADDR=${KRATOS_ADDR}
      - OATHKEEPER_ADDR=${OATHKEEPER_ADDR}
```
but since I don't know why he did that and I don't see a functional reason to do it with `malware-scanner` I prefer to leave the configuration as short as possible

Reply:
it serves as documentation, I kinda like it but it's PP